packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -166,7 +177,9 @@ export interface LoadBalancedServiceBaseProps {
  */
 export abstract class LoadBalancedServiceBase extends cdk.Construct {
   public readonly assignPublicIp: boolean;
-
+  /**
+   * The desired number of instantiations of the task definition to keep running on the service.
```

Comment:
Why only "desiredCount" and "cluster" have comments?

Reply:
This is the first pass, the doc writer will go through the rest after these are updated.